src/index.rst

```diff
@@ -1,28 +1,228 @@
+            arrowsize=0.75,
+        ];
+
+        Augur [fillcolor=1, color=2];
+        Auspice [fillcolor=3, color=4];
+
+        Augur -> Auspice;
+    }
+
+**Augur** is a series of composable, modular bioinformatics tools. We use these
+to create recipes for different pathogens and different analyses, which can be
+reproduced given the same input data and replicated when new data is available.
+
+.. graphviz::
+    :align: center
+
+    graph {
+        graph [
+            ranksep=0.25,
+        ];
+
+        node [
+            shape=box,
+            style="rounded, filled",
+            fontname="Lato, 'Helvetica Neue', sans-serif",
+            fontsize=12,
+            height=0.1,
+            colorscheme=paired10,
+            fillcolor=1,
+            color=2,
+        ];
+
+        edge [
+            arrowhead=open,
+            arrowsize=0.75,
+        ];
+
+        Augur -- ellipsis1 [style=invis];
+        ellipsis1 [label="…", shape=plain, style=""];
+
+        Augur -- {filter, align, tree, refine, export};
+
+        Augur -- ellipsis2 [style=invis];
+        ellipsis2 [label="…", shape=plain, style=""];
+    }
+
+**Auspice** is a web-based visualization program, to present and interact with
+phylogenomic and phylogeographic data. Auspice is what you see when, for
+example, you visit `nextstrain.org/mumps/na
+<https://nextstrain.org/mumps/na>`__.
+
+.. figure:: images/mumps.png
+
+    Auspice displaying Mumps genomes from North America.
+
+Augur produces :doc:`dataset files </reference/formats/data-formats>` which are
+visualized by Auspice.  These files are often referred to as "JSONs"
+colloquially because they use a generic data format called JSON.
+
+.. graphviz::
+    :align: center
+
+    digraph {
+        graph [
+            rankdir=LR,
+        ];
+
+        node [
+            shape=box,
+            style="rounded, filled",
+            fontname="Lato, 'Helvetica Neue', sans-serif",
+            fontsize=12,
+            height=0.1,
+            colorscheme=paired10,
+        ];
+
+        edge [
+            arrowhead=open,
+            arrowsize=0.75,
+        ];
+
+        Augur [fillcolor=1, color=2];
+        Auspice [fillcolor=3, color=4];
+        jsons [label="mumps_na.json\lmumps_na_root-sequence.json"];
+
+        Augur -> jsons -> Auspice;
+    }
+ 
+**Builds** are recipes of code and data that produce result **datasets** for
+visualization and analysis.  Builds run dozens of commands and often use
+`Snakemake <https://snakemake.readthedocs.io>`__ to :doc:`manage the pipeline
+workflow </guides/bioinformatics/augur_snakemake>`, but any workflow system can
+be used, such as `Nextflow <https://nextflow.io>`__ or `WDL
+<https://openwdl.org>`__.  As an example, our core builds are organized as `Git
+repositories <https://git-scm.com>`__ hosted on `GitHub
+<https://github.com/nextstrain>`__ which contain a Snakemake workflow using
+Augur, configuration, and data.
+
+.. graphviz::
```

Comment:
This embedded graphviz is an excellent standard to set for our documentation. I would love to use this approach to replace [the outdated ncov workflow diagram](https://docs.nextstrain.org/projects/ncov/en/latest/analysis/orientation-workflow.html) now that it's in RTD.